keps/sig-node/24-apparmor/kep.yaml

```diff
@@ -0,0 +1,48 @@
+title: Add AppArmor Support
+kep-number: 24
+authors:
+  - "@saschagrunert"
+  - "@tallclair"
+owning-sig: sig-node
+participating-sigs:
+  - sig-api-machinery
+  - sig-auth
+  - sig-security
+status: provisional
```

Comment:
Should be switched to implementable (which will force you to do a prr review and add appropriate prod-readiness file).